{
  try {
    undo.redo();
  }
 catch (  CannotRedoException e) {
    logger.error("Unable to redo.",e);
  }
}
